src/pages/patientView/clinicalInformation/lib/clinicalAttributesUtil.js

```diff
@@ -202,12 +208,28 @@ function getSpanElementsFromCleanData(
     let spans = [];
     return Object.keys(clinicalAttributesCleanDerived).map(key => {
         let value = clinicalAttributesCleanDerived[key];
+        let attrValue = value;
+        // update survival data
```

Comment:
can you improve this comment to explain bussiness log better?  also, i think the following logic should have a test.